### PR TITLE
Update Start-SPOUserAndContentMove.md

### PR DESCRIPTION
**Pull request by AdwoodMicrosoft**
Updated the system reflection for correctness.

### CODE REVIEW

**Comment by dariomws** [comment]
I tested -Reserved parameter and it's available. I think it's better if we add "This parameter is reserved for internal Microsoft use" as description because customers usually ask for some documentation if they see it.

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOUserAndContentMove.md [hunk 1]
```diff
@@ -26,39 +26,39 @@ Start-SPOUserAndContentMove [-UserPrincipalName] <String> [-DestinationDataLocat
 
 ## DESCRIPTION
 
-In a multi-geo company, this cmdlet requires a multi-geo tenant to work, and enables the ability to move a user closer to their sites
+This commandlet applies to Multi-Geo tenants and it is designed to move user profiles and their corresponding OneDrive personal sites across geo locations. These commandlets may only be ran by a SharePoint or Global administrator, who is connected to the SharePoint administration center of the geo location where the user is currently hosted.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOUserAndContentMove.md:29]:
```suggestion
This cmdlet applies to Multi-Geo tenants and it is designed to move user profiles and their corresponding OneDrive personal sites across geo locations. These cmdlets may only be ran by a SharePoint or Global administrator, who is connected to the SharePoint administration center of the geo location where the user is currently hosted. 
```

**Comment by get-itips** [comment]
@AdwoodMicrosoft Adriana, can you kindly apply the suggestions? Thank you.

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft gentle reminder, please apply the suggestions. Thanks.

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft kind request, please apply the suggestions. Thanks.

**Comment by yogkumgit** [comment]
@AdwoodMicrosoft please apply the suggestions, PR is pending to be merged. Thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOUserAndContentMove.md [hunk 2]
```diff
@@ -177,7 +160,7 @@ Accept wildcard characters: False
 
 ### -ValidationOnly
 
-Use this parameter to validate if the user is able to be moved.
+Use this parameter to validate if the user is able to be moved. This parameter is recommended for any user move.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOUserAndContentMove.md:163]: Can you provide some text to explain why this switch is recommended?

**Review by tseward — CHANGES_REQUESTED** [review]

**Pull request merged**